src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
```

Comment:
The package declaration should not have an empty line following it. Please remove the redundant empty line after the package declaration.